lib/action_subscriber/configuration.rb

```diff
@@ -69,11 +69,9 @@ def self.configure_from_yaml_and_cli(cli_options = {}, reload = false)
           absolute_config_path = ::File.expand_path(::File.join("config", "action_subscriber.yml"))
           if ::File.exists?(absolute_config_path)
             erb = ::ERB.new(::File.read(absolute_config_path)).result
-            if defined?(SafeYAML)
-              yaml_config = ::YAML.load(erb, :safe => true)[env]
-            else
-              yaml_config = ::YAML.load(erb)[env]
-            end
+            # Defined in Psych 3.2+ and the new canonical way to load trusted documents:
+            # https://github.com/ruby/psych/issues/533#issuecomment-1019363688
+            yaml_config = ::YAML.unsafe_load(erb)[env]
```

Comment:
Alternatively, we could check to see if `unsafe_load` is defined and not require a minimum version of Psych, but since Psych is a part of the standard library, I figured requiring a minimum of 3.3.2 would be fine.